CHANGELOG.rst

```diff
@@ -31,6 +32,8 @@ Added
 
 Fixed
 ~~~~~
+
+- (`#55 https://github.com/iiasa/climate-assessment/pull/55`_) Pinned requirements to make install more reproducible for users
```

Comment:
I'd extend the changelog a little bit. You introduced poetry and ruff, moved the workflows to python 3.11 added a pre-commit hook, fixed the docs build and probably some other things that I'm forgetting.

Reply:
Nice will do

Reply:
I also added the infiller fetching using ixmp4